CHANGELOG.md

```diff
@@ -0,0 +1,1340 @@
+# Changelog
+
+## [3.1.2](https://github.com/AntiMicroX/antimicrox/tree/3.1.2) (2020-09-30)
+
+[Full Changelog](https://github.com/AntiMicroX/antimicrox/compare/3.1.1...3.1.2)
+
+**Fixed bugs:**
+
+- Right click - Turbo and Clear not working properly [\#7](https://github.com/AntiMicroX/antimicrox/issues/7)
+- Wrong order for setting sets [\#6](https://github.com/AntiMicroX/antimicrox/issues/6)
+
+**Closed issues:**
+
+- Docker Hub upload - or removing Docker [\#50](https://github.com/AntiMicroX/antimicrox/issues/50)
+- Please release a Win Installer/.exe [\#42](https://github.com/AntiMicroX/antimicrox/issues/42)
+- \[Question\] Is "antimicro" profiles is compatible with antimicrox ? [\#41](https://github.com/AntiMicroX/antimicrox/issues/41)
+- I can not install from Fedora repositories ! [\#40](https://github.com/AntiMicroX/antimicrox/issues/40)
+- Fedora resubmission [\#29](https://github.com/AntiMicroX/antimicrox/issues/29)
+- Flatpak packaging [\#9](https://github.com/AntiMicroX/antimicrox/issues/9)
+
+**Merged pull requests:**
+
+- refactor: remove Dockerfiles [\#52](https://github.com/AntiMicroX/antimicrox/pull/52) ([gombosg](https://github.com/gombosg))
+- docs: flatpak readme [\#51](https://github.com/AntiMicroX/antimicrox/pull/51) ([gombosg](https://github.com/gombosg))
```

Comment:
I think we should also avoid putting `docs`, into changelogs, becasue things like updating readme doesn't change anything for end-user

Reply:
If we have a 'Merged pull requests' section then it doesn't make sense to list only _some_ of them. So either I remove this section as well (but then why are we generating it? :slightly_smiling_face: ) or keep it as is.

Reply:
Maybe we should have just one section: `changes` and we would put there all of viable updates.

Currently, we have duplicates for fixes (Issue + PR).

Reply:
I thought that Conventional Commits will be used for generating these changelogs, and we changelog would be generated based mainly on them.

Reply:
Ok then I'll try something like [this](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli) tomorrow